File: library/src/main/java/me/shkschneider/skeleton/helper/ClipboardHelper.java
package me.shkschneider.skeleton.helper;

import android.content.ClipData;
import android.support.annotation.NonNull;

public class ClipboardHelper {

    protected ClipboardHelper() {
        // Empty
    }

    public static void copy(@NonNull final String string) {
        final ClipData clipData = ClipData.newPlainText(ApplicationHelper.packageName(), string);
        SystemServices.clipboardManager().setPrimaryClip(clipData);
    }

}

File: library/src/main/java/shkschneider/skeleton/helper/ClipboardHelper.kt
package me.shkschneider.skeleton.helper

import android.content.ClipData

object ClipboardHelper {

    fun copy(string: String) {
        val clipData = ClipData.newPlainText(ApplicationHelper.packageName(), string)
        SystemServices.clipboardManager()?.primaryClip = clipData
    }

}
